### PR TITLE
🐦[FSharp] Falco - v2.1.0

### PR DESCRIPTION
**Pull request by pimbrouwers**
Update to Falco 2.1.0

### CODE REVIEW

File: fsharp/falco/web.fsproj [hunk 1]
```diff
@@ -3,10 +3,11 @@
     <TargetFramework>net5.0</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Include="App.fs" />
     <Compile Include="Program.fs" />
   </ItemGroup>  
   <ItemGroup>
     <PackageReference Update="FSharp.Core" Version="5.0.0" />
```
Comment (waghanza) [inline fsharp/falco/web.fsproj:10]: should we not use `5.0.*` ?

File: fsharp/falco/web.fsproj [hunk 2]
```diff
@@ -3,10 +3,11 @@
...
-    <PackageReference Include="Falco" Version="2.0.*" />
+    <PackageReference Include="Falco" Version="2.1.0" />
```
Comment (waghanza) [inline fsharp/falco/web.fsproj:11]: should we not use `2.1.*` ?

Comment (pimbrouwers) [reply fsharp/falco/web.fsproj]: Apologies. I thought you were requesting me to remove the floating minor version.

Comment (waghanza) [reply fsharp/falco/web.fsproj]: up to you ... if you do not plan to add **breaking changes** in patch level 😛

**Comment by pimbrouwers** [comment]
I would never do such a thing 🙄

**Pull request merged**